samples/openthread/libraries/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+#
```

Comment:
@MarekPorwisz suggested that instead of creating this artificial sample we should rather supplement the CLI example with the functionaity of building and replacing the libraries. I will do it.